editor.planx.uk/src/pages/FlowEditor/components/forms/FormModal.tsx

```diff
@@ -54,6 +54,7 @@ const NodeTypeSelect: React.FC<{
       <optgroup label="Inputs">
         <option value={TYPES.TextInput}>Text Input</option>
         <option value={TYPES.FileUpload}>File Upload</option>
+        <option value={TYPES.MultipleFileUpload}>Multiple File Upload</option>
```

Comment:
nit: Could we comment this out for now, juuuuuust incase, as we don't have validation to catch these "in progress" nodes on Publish.

Reply:
done!